spot-client/src/spot-tv/ui/views/meeting.js

```diff
@@ -221,7 +225,9 @@ export class Meeting extends React.Component {
      */
     _onMeetingLeave(leaveEvent = {}) {
         if (leaveEvent.error) {
-            this.props.dispatch(addNotification('error', leaveEvent.error));
+            this.props.onError(leaveEvent.error);
+        } else {
+            this.props.disconnectAllTemporaryRemotes();
```

Comment:
How about always trying to disconnect remotes? This operation really should be done by the backend, but since the TV tries to act like one maybe let it always give it a try?

Reply:
This is not important and I'm not sure about it. I've been wondering if it would cause problem when the network is offline.

Reply:
Do you mean also trying to kick when there is an error joining? I thought it would be annoying to have to re-pair due to an error on join.

If you mean always kicking after a proper hangup, the open source flow does not guarantee the server is an admin, so there could be inconsistent kick behavior. If you're okay with I dont mind attempting the kick.